src/components/Profile/MyPosts/Post/Post.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+import s from './Post.module.css';
+
+const Post = (props) => {
+    return (
+        <div className={s.item}>
+            <img src='https://yt3.ggpht.com/a/AGF-l79zTH00bozogFe1oAjQyY7-lnxI-uZMKgonrw=s900-mo-c-c0xffffffff-rj-k-no'></img>
+            {props.message}
+            <div>
+                <span>like</span>
```

Comment:
Consider storing "like" in a separate constant, preferably an object - if you have multiple strings like this in your app.

Reply:
Yes, I will do this in future versions of the app.